sentry_sdk/integrations/celery.py

```diff
@@ -107,8 +107,10 @@ def sentry_build_tracer(name, task, *args, **kwargs):
         trace.build_tracer = sentry_build_tracer
 
         from celery.app.task import Task  # type: ignore
+        from celery import Celery  # type: ignore
 
         Task.apply_async = _wrap_apply_async(Task.apply_async)
+        Celery.send_task = _wrap_apply_async(Celery.send_task)
```

Comment:
Can we rename the wrapper to something else now that we're not just using it to wrap `apply_async`? Something more generic, maybe `_wrap_run_task`?